metadata.json

```diff
@@ -69,6 +75,7 @@
     }
   ],
   "dependencies": [
-    {"name":"puppetlabs/stdlib","version_requirement":">=3.2.0 <5.0.0"}
+    {"name":"puppetlabs/stdlib","version_requirement":">=3.2.0 <5.0.0"},
+    {"name":"zleslie/pkgng","version_requirement":">=0.2.5 <2.0.0"}
```

Comment:
Yay, pkgng is in core. Could you remove this dependency, document it in the readme as a soft requirement for puppet < 4.1.0, and that it will work on 4.1.0 as-is?